src/main/java/core/basesyntax/DefaultDataOperationStrategy.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class DefaultDataOperationStrategy implements DataOperationStrategy {
+    @Override
+    public void execute(FruitTransaction transaction, FruitDB fruitDB) {
+        String type = transaction.getType();
+        String fruit = transaction.getFruit();
+        int quantity = transaction.getQuantity();
+
+        switch (type) {
+            case "b":
+            case "s":
```

Comment:
The cases 'b' and 's' both call `fruitDB.add(fruit, quantity);`. Ensure that this is the intended behavior for both transaction types. If they are meant to perform different operations, consider separating their logic.